README.md

```diff
@@ -82,9 +82,10 @@ Torchvision currently supports the following image backends:
 - [libjpeg](http://ijg.org/) - can be installed via conda `conda install jpeg` or any of the package managers for
   debian-based and RHEL-based Linux distributions. [libjpeg-turbo](https://libjpeg-turbo.org/) can be used as well.
 
-**Notes:** `libpng` and `libjpeg` must be available at compilation time in order to be available. Make sure that it is
-available on the standard library locations, otherwise, add the include and library paths in the environment variables
-`TORCHVISION_INCLUDE` and `TORCHVISION_LIBRARY`, respectively.
+**Notes:** `libpng` and `libjpeg` are optional dependencies. If any of them is available on the system, 
```

Comment:
It's good to clarify that libjpeg and libpng are optional, but I don't think we need to remove that sentence:

> `libpng` and `libjpeg` must be available at compilation time in order to be available.

Reply:
Sorry, I do not understand what you ask. Libs are optional and this sentence "libpng and libjpeg must be available at compilation time in order to be available." does not make sense (without speaking that it is unclear what it is exactly "in order be available")

Reply:
I think the sentence means

> libjpeg and libpng need to be available at compile time when you build from source.

We don't need to remove it